client/strings/en-us.json

```
@@ -282,6 +284,7 @@
   "LabelRecentSeries": "Recent Series",
   "LabelRegion": "Region",
   "LabelReleaseDate": "Release Date",
+  "LabelRemoveCover": "Remove book cover",
```

```suggestion
  "LabelRemoveCover": "Remove cover",
```
This is also used for podcasts